cmd/lakectl/cmd/root.go

```diff
@@ -148,6 +124,8 @@ func init() {
 	cobra.OnInitialize(initConfig)
 	rootCmd.PersistentFlags().StringVarP(&cfgFile, "config", "c", "", "config file (default is $HOME/.lakectl.yaml)")
 	rootCmd.PersistentFlags().BoolVar(&noColorRequested, "no-color", false, "don't use fancy output colors (default when not attached to an interactive terminal)")
+	baseURIDefault := os.Getenv("LAKECTL_BASE_URI")
+	rootCmd.PersistentFlags().StringVarP(&baseURI, "base-uri", "", baseURIDefault, "base URI used for lakeFS address parse")
```

Comment:
Can `base-uri` include anything? Just the repo? repo+ref? path too?

Reply:
yes, it can even just include the schema - `--base-uril lakefs://` which is cool